test/features/kogito_service_performance.feature

```diff
@@ -73,33 +73,33 @@ Feature: Kogito Service Performance
       }
       """
 
-    Then HTTP GET request on service "process-quarkus-example" with path "orders" should return an array of size <requests> within 1 minutes
-    And HTTP GET request on service "process-quarkus-example" with path "orderItems" should return an array of size <requests> within 1 minutes
+    #Then HTTP GET request on service "process-quarkus-example" with path "orders" should return an array of size <requests> within 1 minutes
+    #And HTTP GET request on service "process-quarkus-example" with path "orderItems" should return an array of size <requests> within 1 minutes
     #And All human tasks on path "orderItems" with path task name "Verify_order" are successfully "completed" with timing "true"
 
 
     Examples:
       | native   | minutes | requests |
-      | disabled | 10      | 40000    |
-      | disabled | 10      | 80000    |
+#      | disabled | 10      | 40000    |
```

Comment:
is this expected to be commented in/out when needed?

Reply:
I am now still executing tests and use this branch for it. So this will be changed in the final PR.

Reply:
ok, thanks